spec/models/player_list_spec.rb

```diff
@@ -1,5 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe PlayerList, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:x) { "Player X" }
+  let(:o) { "Player O" }
+
+  let(:p1) { PlayerList::Player.new("x", "Player X") }
+  let(:p2) { PlayerList::Player.new("o", "Player O") }
+  describe "players" do
+    context "new" do
+      it do
+        pl = PlayerList.new(x, o, 0)
+        expect(pl.players).to eq [p1, p2]
+        expect(pl.current_player).to eq 0
+        expect(pl.current).to eq p1
+      end
+    end
+
+    context 'next' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.